src/styles/home.scss

```diff
@@ -167,7 +167,7 @@
       }
 
       .Lab__home__introSlider__cardItem__card {
-        height: pxToRem(344px);
+        height: pxToRem(400px);
```

Comment:
would `fit-content` work here?

Reply:
No, because we don't want the height to change when we open the larger cards or when we scroll through them.